AltUseVendors/__init__.py

```diff
@@ -405,8 +421,8 @@ def PerformedSecondaryUseAction(self, caller: unrealsdk.UObject, function: unrea
         info.purchase_function(caller, vendor)
 
         # Play the dialogue sound if one is given for this vendor
-        if vendor.InteractiveObjectDefinition.name in AKE_DICT[Game.GetCurrent()]:
-            vendor.PlayAkEvent(unrealsdk.FindObject("AkEvent", AKE_DICT[Game.GetCurrent()][vendor.InteractiveObjectDefinition.name]))
+        dialogEvent = AKE_VOICELINE_BY_VENDOR.get(vendor.InteractiveObjectDefinition.Name)
+        find_and_play_akevent(vendor, dialogEvent)
```

Comment:
`dialogEvent` (pep8) may be None, but you defined the second arg to `find_and_play_akevent` as just `str`, so you're not allowed to pass that. Need an explicit check for it here - which we can also use to print a debug message in case we haven't covered all vendor types.